.github/workflows/release.yml

```diff
@@ -50,13 +50,9 @@ jobs:
         uses: docker/metadata-action@818d4b7b91585d195f67373fd9cb0332e31a7175
         with:
           images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
-          tags: |
-            type=ref,event=branch
-            type=ref,event=tag
```

Comment:
will we be able to obtain the tag during docker build with this removed ?

`steps.meta-init.outputs.tags`